### DISCUSSION TITLE
BasicScore - REMOVED

### DISCUSSION THREAD

**Discussion by Ifera** [Ifera/ScoreHudX#13] [Tags by the community]
* Name: BasicScore
* Description: Adding basic tags for ScoreHud
* Author: Ifera
* API: 3.0.0
* Version: 1.0.1
* Link: https://github.com/Ifera/ScoreHudX/BasicScore
* Required Plugin: `nil`

**Comment by matrax123** [comment]
> 

Link invalid

**Comment by Ifera** [comment]
it has been removed. BasicScore has been integrated within ScoreHud. You don't have to add this separately in ScoreHud v7 onwards.
